dashboard/src/actions/authActions.js

```diff
@@ -0,0 +1,122 @@
+import * as types from "./types";
+import API from "../utils/api";
+import * as API_ROUTES from "../utils/apiConstants";
```

Comment:
Some style consistency might be nice; here you're importing the `apiConstants` using an all-caps `API_ROUTES`, but in `App.js` you imported the similar set of `routeConstants` as a camel-case `AppRoutes` and `types` is imported using lowercase `types`. Can we choose one style?

Reply:
As suggested, changed the imports to all-caps format.